sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -277,7 +277,7 @@ def _settle_message(self, settlement, lock_tokens, dead_letter_details=None):
         )
 
     def _renew_locks(self, *lock_tokens):
-        # type: (*str) -> Any
+        # type: (str) -> Any
```

Comment:
`*lock_token` could be more than just a single string, right?
so it could be `_renew_lock(token1, token2, token3, token4, ...)`, i'm wondering the type hint here `# type: (str) -> Any` correctly reflects that the method can accept multiple strings.

Reply:
https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html

I had the same question as you; but you'll find that some of the tools (ala pyright) actually complain about this and mypy best practices is to not use *.  I assume they can infer it from the signature itself.

Reply:
cool, I see the `*args` part in the cheat_sheet, thanks for the ref